sdk/storage/Azure.Storage.Blobs/tests/SessionRecords/BlobBaseClientTests/BlobDownloadInfo_DisposeAsync.json

```diff
@@ -0,0 +1,4 @@
+{
+  "Entries": [],
```

Comment:
Why are these empty? Should they be in recorded test class?

Reply:
We have a few tests in the recorded test class that don't actually do any requests so the recordings are empty. It is a tradeoff between needing to create a one-off separate test class vs. having these empty recording files.

Reply:
It's up to you to decide. For me it's slightly cleaner to have a separation between unit and functional tests.

Reply:
Yeah, I guess I would tend to agree.